app/views/exam_templates/_template_division.html.erb

```diff
@@ -5,11 +5,11 @@
                         'aria-label': TemplateDivision.human_attribute_name('label') %>
   </td>
   <td>
-    <%= form.number_field :start, min: 1, disabled: disabled,
+    <%= form.number_field :start, min: 2, disabled: disabled,
```

Comment:
This is good, but you should also modify the `add_template_division` function in `_boot.js` to add the `min` attribute (looks like it wasn't specified at all before).